src/components/menu-top/LoadShow.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="modal-card">
+    <header class="modal-card-head">
+      <p class="modal-card-title">
+        Load Show
+      </p>
+    </header>
+
+    <section class="modal-card-body">
+      <b-field class="file">
+        <b-upload
+          v-model="file"
+          accept=".shw"
+          @input="loadShow"
+        >
+          <a class="button is-primary">
+            <b-icon icon="upload" />
+            <span>Click to load</span>
+          </a>
+        </b-upload>
+        <span
+          v-if="file"
+          class="file-name"
+        >
+          {{ file.name }}
+        </span>
+      </b-field>
+      <span
+        v-if="parseError"
+      >
+        <b-message
+          title="Error"
+          type="is-danger"
+          aria-close-label="Close message"
+        >
+          {{ parseError }}
+        </b-message>
+      </span>
+      <span
+        v-if="show"
+      >
+        <b-field>
+          Marchers: {{ numMarchers }}
+        </b-field>
+        <b-field>
+          Sheets: {{ numSheets }}
+        </b-field>
+      </span>
+    </section>
+
+    <footer class="modal-card-foot">
+      <b-button
+        type="is-success"
+        data-test="file-modal--import"
+        :disabled="!showSet"
+        @click="setShow();$parent.close()"
+      >
+        Import
+      </b-button>
+      <b-button
+        type="is-primary"
+        data-test="file-modal--close"
+        @click="$parent.close()"
+      >
+        Close
+      </b-button>
+    </footer>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { LoadShow } from '@/models/util/LoadShow';
+import Show from '@/models/Show';
+
+export default Vue.extend({
+  name: 'LoadShow',
+  data: (): {
+    file: Blob | null;
+    show: Show | null;
```

Comment:
To prevent confusion with `show` in the global state...
```suggestion
    showPreview: Show | null;
```

Reply:
Done.